WordPress/Classes/ViewRelated/Jetpack/Branding/Banner/JetpackBannerWrapperViewController.swift

```diff
@@ -10,6 +10,11 @@ final class JetpackBannerWrapperViewController: UIViewController {
     /// JPScrollViewDelegate conformance.
     internal var scrollViewTranslationPublisher = PassthroughSubject<Bool, Never>()
 
+    override var navigationItem: UINavigationItem {
+        guard let childVC else { return UINavigationItem() }
+        return childVC.navigationItem
```

Comment:
@hassaanelgarem what do you think of this? It works, but I'm not totally sure it's safe...

Reply:
TODO: Need to update [these calls](https://github.com/wordpress-mobile/WordPress-iOS/pull/19844/files#diff-fa1473480b929ba6777ed89082486079865c179e308dc9b7d167ca24162c8fdeR1627) if we go with this solution.

Reply:
I understand your concern. It is a bit hacky. But I do think it's safe.
Since `navigationItem` is a class (passed by reference), changes done inside the child VC is propagating to the wrapper. And after looking into apple's documentation, it seems common for this property to be overridden.

Reply:
I agree with all of that. Taking into consideration that there's a planned end date for the banner / badge functionality it makes me feel better. 👍